book/docs/Open_formats_tools.ipynb

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
<p>I don't think that Rasterio would be the first choice for most NetCDF or HDF5 files if using Xarray (there are alternate drivers/engines in Xarray). This just a note for later though, not sure any content needs to change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/scaling_science/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>